tests/middleware/test_base.py

```diff
@@ -14,44 +23,59 @@
 from starlette.routing import Route, WebSocketRoute
 from starlette.testclient import TestClient
 from starlette.types import ASGIApp, Message, Receive, Scope, Send
+from starlette.websockets import WebSocket
+
+BytesGenerator = Generator[bytes, None, None]
+AwaitableGenerator = Generator[Any, None, None]
+TestClientFactory = Callable[[ASGIApp], TestClient]
+MiddlewareClass = Type[_MiddlewareClass[Any]]
```

Comment:
Can you instead not create those aliases, please? You can leave the `TestClientFactory` here, but the ones that are used only once, I'd prefer to avoid.

Reply:
Sure, deleted them.